uber/templates/regform.html

```diff
@@ -357,7 +383,7 @@
 </div>
 
 <div class="form-group">
-    <label for="cellphone" class="col-sm-2 control-label">Your Cellphone Number</label>
+    <label for="cellphone" class="col-sm-2 control-label"><b>Your Cellphone Number</b></label>
     <div class="col-sm-6">
```

Comment:
would have to double check, but I think this is an optional field except for volunteers, which is why we left it un-bolded.